src/components/Quote/Quote.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { styles } from './Quote.styles'
+
+export interface QuoteProps {
+  children?: React.ReactNode
+  cite?: string
+  symbol?: 'default' | 'quote'
+  align?: 'default' | 'middle'
+}
+
+export const Quote = ({ children, cite, symbol = 'default', align = 'default' }: QuoteProps) => {
```

Comment:
- symbol = 'default' would change to = 'border' based on style changes
- align = 'default' would be replace with only isCenter, no default setting is required on a boolean